src/bitcoin-cli.cpp

```diff
@@ -518,7 +518,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
                 const std::string addr{peer["addr"].get_str()};
                 const std::string age{conn_time == 0 ? "" : ToString((time_now - conn_time) / 60)};
                 const std::string sub_version{peer["subver"].get_str()};
-                const std::string transport{peer["transport_protocol_type"].get_str()};
+                const std::string transport{peer["transport_protocol_type"].isNull() ? "" : peer["transport_protocol_type"].get_str()};
```

Comment:
Can't we just default to `"v1"` instead of `""` in case of null? Pre-v26 doesn't support v2 transport, it will always be v1.